.github/workflows/lint.yml

```diff
@@ -0,0 +1,18 @@
+name: Lint
+on: [pull_request]
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Whitespace errors
+      run: |
+        git config core.autocrlf false
```

Comment:
I am not sure it is useful to disable autocrlf for this test when we otherwise actively enable it in `.gitattributes`, is it?

Reply:
You are right, I don't think we need it.